### DISCUSSION TITLE
🔗 🚀Feature Request: Mega Menu - Open On Hover Should Have Delay Value for better UX (Duplicate of #21353)

### DISCUSSION THREAD

**Discussion by lensbreak** [elementor/elementor#23367] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently if you set the Mega Menu items to "Open On: Hover", the dropdown content immediately appears. That can cause an usability problem if the menu has many items and used e.g. in a header. If the user moves the mouse around and accidentially hover over the menu, it opens and covers the website content. If the dropdown content is very large, then its even difficult the move the mouse somewhere to make the menu closing again.

### Describe the solution you'd like

 Ideally you can choose a certain amount delay in (mili)seconds that will pass between hovering the menu item until the dropdown content is being displayed.

### Describe alternatives you've considered

A good example can be found how https://www.bhphotovideo.com/ implemented their mega menu (green header bar). If you quickly cross over the menu items in the green area, nothing happens except the items color chaing to white but no dropdown content appears.  Only if you remain on the menu item for about 0.5s then the content appears. This delay should be added to Elementor Mega Menu and being set by the webdesigner.

**Comment by nicholaszein** [comment]
Hello there, @lensbreak !

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #21353

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
